app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -636,23 +638,44 @@ export default Ember.Component.extend({
     return null;
   },
 
+  @computed
```

Comment:
not sure why this needs to be a property here, are we using this elsewhere than in `paste(e)` ?

if not I would just replace 

```
const placeholder = plainText || this.get('pastePlaceholder');
```

by

```
const placeholder = plainText || I18n.t('pasting');
```

and remove this